src/dc_configure.rs

```diff
@@ -994,7 +994,7 @@ pub unsafe fn dc_job_do_DC_JOB_CONFIGURE_IMAP(context: &Context, _job: *mut dc_j
                                                                                         as
                                                                                         uintptr_t
                                                                                 );
-                                                                                dc_ensure_secret_key_exists(context);
+                                                                                dc_ensure_secret_key_exists(context).ok();
```

Comment:
According to the [documentation](https://doc.rust-lang.org/std/result/enum.Result.html#method.ok), `ok` converts `Result` to `Option`. What is the purpose of doing it here?